Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -194,7 +194,9 @@ private func rewardTitle(project: Project, reward: Reward) -> String {
 private func pillStrings(project: Project, reward: Reward) -> [String] {
   var pillStrings: [String] = []
 
-  if let endsAt = reward.endsAt, project.state == .live, endsAt > 0,
```

Comment:
Do we we want to remove the check from the `remaining` logic too?

https://github.com/kickstarter/ios-oss/blob/8080091b9f023fb8d1ffb87faec20f217ee1a95e/Library/ViewModels/RewardCardViewModel.swift#L210

Reply:
sure 👍